integration_test.go

```diff
@@ -0,0 +1,389 @@
+package viamcartographer_test
```

Comment:
For this and the files below, would you mind adding a comment that links to which file(s) in rdk they're largely copied from?

Reply:
Good idea! 

integration_test.go was formerly [cartographer_int_test.go](https://github.com/viamrobotics/rdk/blob/main/services/slam/builtin/cartographer_int_test.go)